wg-policy/README.md

```diff
@@ -20,7 +20,7 @@ Provide an overall architecture that describes both the current policy related i
 * SIG Storage
 
 ## Meetings
-* Regular WG Meeting: [Wednesdays at 16:00 PT (Pacific Time)](https://zoom.us/j/7375677271) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:00&tz=PT%20%28Pacific%20Time%29).
+* Regular WG Meeting: bi-weekly [Wednesdays at 08:00 AM PT (Pacific Time)](https://zoom.us/my/cncfsigsecurity) (weekly). [Convert to your timezone](http://www.thetimezoneconverter.com/?t=16:00&tz=PT%20%28Pacific%20Time%29).
```

Comment:
This should be separate from the CNCF group and managed under a kubernetes owned zoom account.

Reply:
Here's where I got that from: https://docs.google.com/document/d/1ihFfEfgViKlUMbY2NKxaJzBkgHh-Phk5hqKTzK-NEEs/edit. What zoom should be used instead?